### PR TITLE
Bump recipe

### PR DESCRIPTION
**Pull request by maxhelias**
After installing the project, I made an update of the recipes with `composer sync-recipes --force`
That's what it gives, tell me if there are things to remove.

Q | A
-- | --
Bug fix? | no
New feature? | no
BC breaks? | no
Deprecations? | no
Tests pass? | yes
Fixed tickets |  
License | MIT
Doc PR

### CODE REVIEW

File: api/public/index.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 }
 
 if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? $_ENV['TRUSTED_PROXIES'] ?? false) {
-    Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_ALL);
+    Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_ALL ^ Request::HEADER_X_FORWARDED_HOST);
```
Comment (dunglas) [inline api/public/index.php]: This must be reverted. It prevents Varnish to work properly.

Comment (maxhelias) [reply api/public/index.php]: It's reverted 😃

**Comment by maxhelias** [comment]
Yes, it's a good idea, I just added it but how to prevent updating the index.php file for Varnish ?
Will not we change the recipe ? Or something else ?

**Comment by dunglas** [comment]
For now will do it manually. Of maybe could we just add a ˋsed` line to remove the extra constant automatically?

**Comment by maxhelias** [comment]
Ok, I understand better now how does the command work.
I delete the file symfony.lock which is used to check the version of the recipe and executed the command to have all the updates (like that of Mercure).
And add the new generated symfony.lock file.

Also the command `composer sync-recipes --force` don't work very well in container php because we don't have the reference Git.

**Comment by maxhelias** [comment]
We don't have the update of index.php file because the symfony.lock file keep the ref and the command check if we have new version of the recipe.
So with the new symfony.lock file, we don't have any probleme with the new update for now

**Comment by dunglas** [comment]
Thank you very much Max!

**Pull request merged**